Module_00/README.md

```diff
@@ -1,4 +1,67 @@
+    - JavaScript (no relation to Java!)
+  - We use certain tools and "compilers" for the browser to then translate what we write into one, two, or all of those three
+  - Then those three get compiled down to even more technical machine code that our computers can read (all dem 0s and 1s)
+
+## Vocabulary
+  Term  | Definition
+  ----  | ----
+  Terminal/Command Line | The tool we use to access the skeleton/bones of our system. We can create, remove, move, edit, etc. files and folders all from within the terminal!
+  HTML  | Hypertext Markup Language
+  CSS | Cascading style sheets
+  JavaScript |  One of the foundational languages that the browser reads (along with HTML and CSS)
+  Markdown  | A language that allows us to mock/create files formatted such as this one without needing a formatting bar (bullet points, tables, bold, italics, etc...)
+  Directory | A folder!
+  Path | Either `relative` or `absolute` the path shows how to get from one file/folder to another
+  Code Snippet | A little 'blurb' of code.  Feel free to write multipline code snippets in SLACK by surrounding your lines with triple-ticks (to the left of the 1 key) or single ticks for single-line snippets (ex: `my first code snippet`)
+  ```
+  multi
+  line
+  snippet
+  ```
+
+## Let's Dig In!
+  - When we code, often times we do so using a structure called "Pair Programming".  This allows us as developers (which you all are now!) to both 'drive' and 'navigate' while coding, essentially eliminating typos and providing immediate feedback to your pair.
+  - Go ahead and join my LiveShare session so we can all pair together!
+
+***
+
+### Intro to Console and HTML (just a bit!)
+  - Open VSCode, and select 'New Terminal' in the 'terminal' menu
+  - Terminal allows us to dig into the skeleton of our computers and create files and folders wherever we'd like in a snap!
+    - To see where you currently are within your system type `pwd` (print working directory) in your terminal/command line
+    - Typing `ls` in your directory will 'list' every file and folder within it
+    - `cd` stands for 'change directory' (that's what we call a folder) and can take a `path` to move from one place to another (`./` indicates your current directory, `../` goes up a directory)
+    - `touch` allows your create a new file directly from your console!
+    - `mv` allows us to rename or move a file or directory
+    - `rm` allows us to remove a file or directory
+    - `history` will show us the recent history of our command line entries
+  - Alrighty, back to it! Using the commands above in your terminal, create a directory called `ThinkCERCAMP` on your Desktop with one file inside called `your_name_00.html`
+  - Open the file in your VSCode editor, and begin typing an exclamation mark `!`.  You will see a prompt appear that leads you to an 'Emmett Abreviation', click tab. Walah! (Emmett is a tool built in to your version of VSCode that provides many shortcuts for commonly used code snippets)
```

Comment:
I found an Emmet plugin for vim! I still don't understand all the snippets, but its pretty cool